core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -150,6 +167,16 @@ import scala.concurrent.duration._
       log.debug("Resetting pid {}. TagProgress {}", pid, tp)
       become(idle(buffer.filterNot(_._1.persistenceId == pid), tagPidSequenceNrs + (pid -> tagPidSequenceNr)))
       sender() ! ResetPersistenceIdComplete
+
+    case ReceiveTimeout =>
+      if (buffer.isEmpty && tagPidSequenceNrs.isEmpty)
+        parent ! PassivateTagWriter(tag)
+
+    case StopTagWriter =>
+      if (buffer.isEmpty && tagPidSequenceNrs.isEmpty)
```

Comment:
tagPidSequenceNrs being empty, so we only stop if actors that have previously written this tag have also stopped?